lib/importeur/data_sources/rocketfuel.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Importeur
+  module DataSources
+    class Rocketfuel
+      def initialize(rocketfuel_service)
+        @rocketfuel_service = rocketfuel_service
```

Comment:
See my suggestion in https://github.com/ad2games/importeur/pull/1/files#r138452203. So we could get rid of that `and_wrap_original` in the test.